### PR TITLE
LPS-193674 [MARKETPLACE] Add responsiveness to header

### PR DESCRIPTION
**Pull request by joanagrego**

### CODE REVIEW

**Comment by joanagrego** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/public-sites-navigation/fragments/mobile-menu-buttom/index.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.show-menu {
+	height: 200px;
+	opacity: 1;
+	transition: height 0.6s , opacity 0.8s;
+}
+
+.hidden-menu {
+	height: 0px;
```
Comment (welitonpe) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/public-sites-navigation/fragments/mobile-menu-buttom/index.css:8]: If any CSS property is 0 it is not necessary to put the type of measurement unit (px, rem, vh...)

```suggestion
	height: 0;
```

**Review by welitonpe — APPROVED** [review]
Good Job :fire: :fire: :fire:

**Review by juliocezar96 — APPROVED** [review]
Nice job! 🤙

**Review by claricebrasil — APPROVED** [review]
Good job!

**Comment by joanagrego** [comment]
ci:test:relevant

**Comment by joanagrego** [comment]
ci:test:relevant

**Comment by claricebrasil** [comment]
ci:forward

**Comment by joanagrego** [comment]
ci:stop

**Comment by joanagrego** [comment]
Just starting reviewing :)

**Comment by joanagrego** [comment]
Resent --> [PR](https://github.com/pt-liferay-solutions/liferay-portal/pull/4702)

**Pull request closed without merging**